rancher2/schema_cluster_eks_config.go

```diff
@@ -37,6 +37,7 @@ type AmazonElasticContainerWorkerPool struct {
 
 	AMI                         string   `json:"ami,omitempty" yaml:"ami,omitempty"`
 	AssociateWorkerNodePublicIP *bool    `json:"associateWorkerNodePublicIp,omitempty" yaml:"associateWorkerNodePublicIp,omitempty"`
+	CreatePoolPerSubnet         bool     `json:"createPoolPerSubnet" yaml:"createPoolPerSubnet"`
```

Comment:
Don't we need to use `*bool` when default is true?

Reply:
@dstockton if you check the json annotation, I haven't added `omitempty` to it so `false` value will be set to rancher instead of omitting the field